tokens.xml

```diff
@@ -4086,6 +4187,21 @@ When this creature enters the battlefield, it deals 1 damage to any target.</tex
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Illusion    </name>
```

Comment:
You must add another space at the end of this token's name, because the previous token already ends in four spaces.